File: datasets/IR-Plag/convertai-original-java-rest-python/case-05/non-plagiarized/12/T05.py
import sys

def reverse(number):
    reverse = 0
    while number != 0:
        reverse = reverse * 10
        reverse = reverse + number % 10
        number = number // 10
    print(reverse)

if __name__ == "__main__":
    a = int(input("Enter an integer: "))
    reverse(a)

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-05/non-plagiarized/06/T05.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function reverse(number: number): void {
  const numberString: string = number.toString();
  let reversedString: string = '';
  for (let i = numberString.length - 1; i >= 0; i--) {
    reversedString += numberString[i];
  }
  console.log(reversedString);
}

rl.question('Enter an integer: ', (input) => {
  const number: number = parseInt(input);
  reverse(number);
  rl.close();
});

